### DISCUSSION TITLE
How do I create a 3 level nested object with the deepest one referencing to top level id (auto-generated)?

### DISCUSSION THREAD

**Discussion by sfabriece** [prisma/prisma#5655] [Q&A]
In my project, id's are auto generated in the database using: `uuid_generate_v4()`.

I want to be to created a record with multi-level cascades. something like: 

```
await prisma.firstLevel.create({
  data: {
    secondLevel: {
      create: {
        thirdLevel: {
          create: {
            someValue: 'x'
          }
        }
      }
    }
  }
});
```

`ThirdLevel` table in the database has a foreign key to both `secondLevel` and `firstLevel`. I would rather not create the Id in the backend and let the database handle that.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
